docs/SRS-IEEE/SRS.tex

```diff
@@ -71,28 +72,20 @@ \subsection{Definitions, Acronyms and Abbreviations}
 \label{sub:def_acr_abb}
 
 \begin{itemize}
-    \item \labeltext{AR object}{def:ar_obj}: A 2D/3D projection of an entity.
+    \item \labeltext{AR object}{def:ar_obj}: A 2D/3D model representing an object that can be presented in AR
+          \begin{itemize}
+              \item \labeltext{Artifact}{def:artifact}: a 3D AR Object
+              \item \labeltext{Sticker}{def:sticker}: a 2D AR Object
+          \end{itemize}
+    \item \labeltext{AR object instance}{def:ar_obj_instance}: An instance of an AR object placed in the real world, defined by an AR object, and a location
+    \item \labeltext{AR object cluster}{def:ar_obj_cls}: A group of AR objects present within a confined area.
     \item \labeltext{Users}{def:user}: A term for anyone who uses the app.
-    \item \labeltext{Organization users}{def:org_user}: Users who belong to an organization that has the ability to create tours. They are affiliated with a particular approved organization who have the ability to modify tours within their organization’s domain.
+    \item \labeltext{Organization users}{def:org_user}: Users who belong to an organization that have the ability to create tours. They are affiliated with a particular approved organization who have the ability to modify tours within their organization’s domain.
     \item \labeltext{General users}{def:gen_user}: Users that can access most of the app functionalities except creating tours.
     \item \labeltext{Admins}{def:admin}: People who have access to nothing else but the admin interface within the app.
-    \item \labeltext{AR object clusters}{def:ar_obj_cls}: A group of AR objects present within a confined area.
-\end{itemize}
-
-\subsubsection{Screen Names}
```

Comment:
Why did you remove the screen names' definitions?

Reply:
it was moved to the user interfaces section and renamed interface names or something

Reply:
Resolved